ovos_plugin_manager/pipeline.py

```
@@ -19,3 +26,116 @@ def load_pipeline_plugin(module_name: str) -> type(PipelinePlugin):
+                    matchers.append((pipe_id, m.match_low))
+            elif isinstance(m, PipelineStageMatcher) and not skip_stage_matchers:
+                matchers.append((pipe_id, m.match))
+        return matchers
+
+    @staticmethod
+    def shutdown():
+        for pipe in OVOSPipelineFactory._CACHE.values():
+            if hasattr(pipe, "shutdown"):
+                try:
+                    pipe.shutdown()
+                except:
```

Ruff

Do not use bare `except`

(E722)